app/components/TrackComponent/index.js

```diff
@@ -65,36 +68,47 @@ export const TrackComponent = memo(function TrackComponent({
     pauseTrack(audioRef);
   };
 
+  // eslint-disable-next-line no-shadow
+  const handleTrackDetailsRoute = (trackId) => {
+    history.push(`/itunes/${trackId}`);
+  };
+
```

Comment:
Why eslint is disabled ?

Also can be written like the below
const handleTrackDetailsRoute = (trackId) => history.push(`/itunes/${trackId}`);

Reply:
fixed!